services/database/serverless.yml

```diff
@@ -143,3 +172,15 @@ resources:
       Value: !Ref McparFormBucket
     McparFormBucketArn:
       Value: !GetAtt McparFormBucket.Arn
+        McparReportTableName:
```

Comment:
Indentation here is gonna break

Reply:
I also think you mean MlrReportTableName & the Value portion should reference the MrReportTable